### ISSUE TITLE
Support - `Cauta` button

### ISSUE THREAD

**Issue by viktorovich87** [asd95/tt-adminui#7]
При клике на кнопку Cauta на странице Support, когда поля пустые - никаких оповещени под инпутами не показывается, что форма заполнена некорректно

**Comment by asd95** [comment]
Согласен, моя вина. Просто забыл добавить функцию с этим. Ибо как сейчас смотрю обработка сама есть, но она почему-то добавлена как комментарий для одного поля.
Я бы сделал так. Добавил стэйт err, и сделал бы условие в UseEffect, чтобы она изменила поле err = true. И при перерендере вывести ошибку.
Спасибо что заметили. Я об этом почему-то забыл(

**Comment by viktorovich87** [comment]
у Textfield есть пропс такой errorText - с ним надо работать

**Comment by asd95** [comment]
Аххх
90 строчка в Support  у input  // helperText={errors?.idnp?.message}
да, видимо реально чет затупил и не сделал.
